Berlin/PetermannIINachtrag42.xml

```diff
@@ -48,183 +147,419 @@ schematypens="http://relaxng.org/ns/structure/1.0"?>
                         </support>
                         <extent>
                            <measure unit="leaf">298</measure>
+                           <measure unit="g" type="weight">5100</measure>
+                           <dimensions type="outer" unit="mm">
+                              <height>300</height>
+                              <width>220</width>
+                                  <depth>105</depth>
+                           </dimensions>
                         </extent>
+                        <foliation>Foliation in the outer upper corner of each recto. Two paper guard leaves each at the beginning and end of the manuscript,
+                           part of the European binding, 
+                        have not been foliated and are referred to here as <locus from="i" to="iv"/>.</foliation>
+                        <collation>
+                           <note>A quaternion is missing between <locus target="#63"/> and <locus target="#64"/>, of which
+                           two folia are located at <locus target="#293"/> and <locus target="#297"/>. <locus target="#296"/> should be located
+                           before <locus target="294"/>.</note>
+ 
+                              <signatures>Undecorated quire signatures in red and black in the upper inner corner of the first leaf of 
+                              <ref target="#q5"/> to <ref target="#q7"/> and <ref target="#"/> to <ref target="#"/>. The quire numeration
+                              starts anew with <ref target="#"/>, with the first quire mark being <foreign xml:lang="gez">፪</foreign> on <locus target="#"/>.</signatures>
+                              <list>
+                                 <item xml:id="q1" n="A">
+                                    <dim unit="leaf">1</dim>
+                                    <locus from="1r" to="1v"/>
+                                 </item>
+                                 <item xml:id="q2" n="1">
+                                    <dim unit="leaf">8</dim>
+                                    <locus from="3r" to="9v"/>
```

Comment:
What about folio 2? Is it missing or did I miss a note on that irregularity? 3r to 9v means just only 7 leaves, but not 8.